test/embag_test_python.py

```diff
@@ -201,5 +201,12 @@ def testDictUnpacking(self):
             assert isinstance(msg.dict()['pose']['covariance'], memoryview if sys.version_info >= (3,3) else np.ndarray)
             assert isinstance(msg.dict(types_to_unpack={embag.RosValueType.float64})['pose']['covariance'], list)
 
+    def testROSTimeDicting(self):
+        for msg in self.view.getMessages('/base_pose_ground_truth'):
+            assert isinstance(msg.dict()['header']['stamp'], embag.RosTime)
+            assert isinstance(msg.dict(ros_time_py_type=None)['header']['stamp'], embag.RosTime)
```

Comment:
Can we also test `to_sec` and `to_nsec` just to make sure that casting is working fine

Reply:
May be have something like 

```
x = msg.dict(ros_time_py_type=None)['header']['stamp']
assert x.to_sec() == x.to_nsec()*(10**9) == msg.dict(ros_time_py_type=int)['header']['stamp'] == msg.dict(ros_time_py_type=float)['header']['stamp'] * 10**9
```

Reply:
I like it!